src/custom/components/CurrencyInputPanel/CurrencyInputPanelMod.tsx

```diff
@@ -14,6 +14,7 @@ import { ReactComponent as DropDown } from 'assets/images/dropdown.svg'
 import { useActiveWeb3React } from 'hooks'
 import { useTranslation } from 'react-i18next'
 import { WithClassName } from 'types'
+import { DEFAULT_PRECISION } from '@src/custom/constants'
```

Comment:
```suggestion
import { DEFAULT_PRECISION } from 'constants/index'
```

not sure if we want to avoid `@src` when possible